README.md

```diff
@@ -67,71 +78,47 @@ mantle codebase.
+run native go code directly on one of the Container Linux machines. This is
 currently possible by using the `NativeFuncs` field of a kola `Test`
 struct. This like a limited RPC interface.
 
 `NativeFuncs` is used similar to the `Run` field of a registered kola
 test. It registers and names functions in nearby packages.  These
 functions, unlike the `Run` entry point, must be manually invoked inside
 a kola test using a `TestCluster`'s `RunNative` method. The function
-itself is then run natively on the specified running CoreOS instances.
+itself is then run natively on the specified running Container Linux instances.
 
 For more examples, look at the
 [coretest](https://github.com/coreos/mantle/tree/master/kola/tests/coretest)
 suite of tests under kola. These tests were ported into kola and make
 heavy use of the native code interface.
 
-## ore
-
-Related commands to launch instances on Google Compute Engine(gce)
-within the latest SDK image. SSH keys should be added to the gce project
-metadata before launching a cluster. All commands have flags that can
-overwrite the default project, bucket, and other settings.  `ore help
-<command>` can be used to discover all the switches.
-
-### ore upload
-
-Upload latest SDK image to Google Storage and then create a gce image.
-Assumes an image packaged with the flag `--format=gce` is present.
-Common usage for CoreOS devs using the default bucket and project is:
-
-`ore upload`
+### Manhole
+The `platform.Manhole()` function creates an interactive SSH session which can
+be used to inspect a machine during a test.
 
-### ore list-images
-
-Print out gce images from a project. Common usage:
-
-`ore list-images`
-
-### ore create-instances
-
-Launch instances on gce. SSH keys should be added to the metadata
-section of the gce developers console. Common usage:
-
-`ore create-instances -n=3 -image=<gce image name> -config=<cloud config file>`
-
-### ore list-instances
-
-List running gce instances. Common usage:
-
-`ore list-instances`
+## ore
 
-### ore destroy-instances
+Ore provides a low level interface for each cloud provider. It has commands 
+related to launching instances on a variety of plaforms (gcloud, aws,
+azure, esx, and packet) within the latest SDK image. Ore mimics the underlying
+api for each cloud provider closely, so the interface for each cloud provider
+is different. `ore help <provider>` lists the available actions for
+each provider.
 
-Destroy instances on gce based by prefix or name. Images created with
-`create-instances` use a common basename as a prefix that can also be
-used to tear down the cluster. Common usage:
+Note, when uploading to some cloud providers (e.g. gce) the image may need to be packaged
+with a different --format (e.g. --format=gce) when running `image_to_vm.sh`
 
-`ore destroy-instances -prefix=$USER`
+SSH keys should be added to the project metadata before launching a cluster. All commands
```

Comment:
Not all platforms support this, and even on platforms that do, it's probably better to pass userdata that configures an SSH key.